x-pack/plugins/security/server/session_management/session_index.ts

```diff
@@ -81,6 +81,7 @@ export function getSessionIndexSettings({
   indexName: string;
   aliasName: string;
 }): IndicesCreateRequest {
+  // @ts-expect-error
```

Comment:
Nit: I personally prefer to add some text to this comment explaining what we're trying to ignore. At a minimum, I just copy the TS error verbatim if it's not too long